thoth-app/src/models/location/update_location_mutation.rs

```diff
@@ -0,0 +1,62 @@
+use serde::Deserialize;
+use serde::Serialize;
+use thoth_api::model::location::Location;
+use thoth_api::model::location::LocationPlatform;
+use uuid::Uuid;
+
+const UPDATE_LOCATION_MUTATION: &str = "
+    mutation UpdateLocation(
+        $locationId: Uuid!,
+        $publicationId: Uuid!,
+        $landingPage: String,
+        $fullTextUrl: String,
+        $locationPlatform: LocationPlatform!,
+        $canonical: Boolean!
+    ) {
+        updateLocation(data: {
+            locationId: $locationId
+            publicationId: $publicationId
+            landingPage: $landingPage
+            fullTextUrl: $fullTextUrl
+            locationPlatform: $locationPlatform
+            canonical: $canonical
+        }){
+            locationId
```

Comment:
Not all fields need to be retrieved here - e.g. I don't think `createdAt` or `updatedAt` are used in the display

Reply:
As discussed, I retract this comment! :upside_down_face: